dashboard/src/actions/charts.ts

```diff
@@ -156,3 +167,35 @@ export function getChartValues(
     }
   };
 }
+
+export function getChartUpdates(
+  name: string,
+  currentVersion: string,
+  appVersion: string,
+): ThunkAction<Promise<void>, IStoreState, null, ChartsAction> {
+  return async dispatch => {
```

Comment:
Please, remove the `async` if you're not using `await` inside.

Reply:
Actually we need to return a promise here as it's a redux thunk action. Should the promise be explicitly defined? I like the use of `async` as it avoids the nesting.

Reply:
ah, I also missed the await haha :P